t/09_qesapdeployment.t

```diff
@@ -8,6 +8,10 @@ use List::Util qw(any);
 use testapi 'set_var';
 use qesapdeployment;
 set_var('QESAP_CONFIG_FILE', 'MARLIN');
+set_var('BACKEND', 'qemu');
+set_var('AUTOINST_URL_HOSTNAME', 'praisethesun');
+set_var('QEMUPORT', 9000);
+set_var('JOBTOKEN', "solaire");
```

Comment:
Fine for this PR, needs to be changed in the future. Without them the error was:

```
#   Failed test 'No tests run for subtest "[qesap_get_inventory] upper case"'
#   at t/09_qesapdeployment.t line 16.
Could not retrieve required variable WORKER_HOSTNAME at /__w/os-autoinst-distri-opensuse/os-autoinst-distri-opensuse/lib/qesapdeployment.pm line 80.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 2.
t/09_qesapdeployment.t ..... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/2 subtests 
```

Reply:
Ah yes... the thing is if you do the data_url mock function, you will need it almost for all test existing there. 
The other this is that tests like:
'[qesap_get_inventory] upper case'
'[qesap_get_inventory] lower case'
Will kind of only test if perl lc function is working. Do we really test stuff like that? Especially if VR is required for each PR which should cover things like that. 
If you use variables, you will at least check if the functions are behaving correctly with testapi.